dashboard-generator/index.html

```diff
@@ -13,6 +13,19 @@
 
 <script src="https://telemetry.mozilla.org/v2/telemetry.js"></script>
 <script src="generator.js"></script>
+
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-49796218-2"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){
+    dataLayer.push(arguments);
+  }
+  gtag('js', new Date());
+
+  gtag('config', 'UA-49796218-2');
+</script>
```

Comment:
Do you think we could consolidate this snippit into a js file that we import from globally? It seems like we repeat this many times.